sklearn/ensemble/_weight_boosting.py

```diff
@@ -328,6 +328,13 @@ class AdaBoostClassifier(ClassifierMixin, BaseWeightBoosting):
         Thus, it is only used when `base_estimator` exposes a `random_state`.
         Pass an int for reproducible output across multiple function calls.
         See :term:`Glossary <random_state>`.
+    n_samples : int, default = None
```

Comment:
I think that we would call this `max_samples` (in `BaggingClassifier` or `RandomForestClassifier`)